examples/certification/certified-counter/src/frontend/.ic-assets.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
New DFX feature means we can use a template for security headers and only specify our headers that will be different from the template. The template's headers match the deleted file, since those headers were taken from ProdSec's recommended list anyway.